src/components/ToDoForm.vue

```diff
@@ -1,25 +1,25 @@
 <template>
   <form>
-    <input v-model="todo" type="text">
-    <button @click.prevent="handleClick" >Submit</button>
-    <button @click.prevent="emit('clearToDo')">Clear</button>
+    <button @click.prevent="handleClick">Add</button>
   </form>
 </template>
 
 <script setup lang="ts">
-	import { ref } from 'vue';
-  const todo = ref("")
+import { Todo } from '../todo.ts'
```

Comment:
Imports order should be:

1. imports from vue: ref, computed etc
2. general components
3. other components
4. store files
5. helpers
6. types

Please use this order from now on